src/pages/ReportParticipantsPage.tsx

```diff
@@ -139,16 +151,13 @@ function ReportParticipantsPage(props) {
     );
 }
 
-ReportParticipantsPage.propTypes = propTypes;
-ReportParticipantsPage.defaultProps = defaultProps;
 ReportParticipantsPage.displayName = 'ReportParticipantsPage';
 
 export default compose(
-    withLocalize,
-    withReportOrNotFound(),
-    withOnyx({
+    withOnyx<ReportParticipantsPageProps, ReportParticipantsPageOnyxProps>({
```

Comment:
based on [TS guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/TS_STYLE.md#hooks-instead-of-hocs) we shouldn't use `compose`